salsah/src/public/js/jquery.searchbox.js

```diff
@@ -217,6 +224,15 @@
 								$this.css({cursor: 'text'});
 							}
 							localdata.ele.sel.hide();
+							// when leaving this input, if the value is empty, but we had a previous valid input
+							// set back the previous input
+							if ($this.attr("isValid") && $this.attr("isValid") == "false") {
```

Comment:
But I guess in this case you could also write: `$this.attr("isValid") === "false"`: he condition will only be true, if the string "false" is returned.

Probably I am opening Pandora's box here because this kind of discussion could totally get out of hand ;-)